src/constants.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable max-len */
+
+const noTextError = {
```

Comment:
Here you have 3 variables with same structure for text data. Will be better if you create an object, add in this object text props and export only object. Because in future when you want to add some new text data you have to create a new variable, export this variable and import in another file. Many different steps. But if you use an object, you add new prop and export object only one time. You don't need to import smth else because all info in your object.